monero-sys/src/lib.rs

```diff
@@ -396,11 +396,12 @@ impl WalletHandle {
         &self,
         address: &monero::Address,
         amount: monero::Amount,
+        split_change: bool,
```

Comment:
Not major, but in general I prefer enums to bool parameters, in this case the `SplitChange::Yes` variant could take an int that define how many change outputs to create which could then default to 3 and be configurable by the ASB? 

Could always be done in a follow-up as well if desired.

Reply:
I agree 👍